File: 2091-removing-minimum-and-maximum-from-array/2091-removing-minimum-and-maximum-from-array.java
class Solution {
    public int minimumDeletions(int[] nums) {
        int a=0;
        int b=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]<nums[a]){
                a=i;
            }
            if(nums[i]>nums[b]){
                b=i;
            }
        }
         int left=Math.min(a,b);
         int right=Math.max(a,b);
         int front=right+1;
         int back=nums.length-left;
         int both=(left+1)+(nums.length-right);
         return Math.min(front,Math.min(back,both));
    }
}

File: 2091-removing-minimum-and-maximum-from-array/2091-removing-minimum-and-maximum-from-array.py
class Solution:
    def minimumDeletions(self, nums: List[int]) -> int:
        a=0
        b=0
        for i in range(len(nums)):
            if nums[i]<nums[a]:
                a=i
            if nums[i]>nums[b]:
                b=i
        left=min(a,b)
        right=max(a,b)
        front=right+1
        back=len(nums)-left
        both=(left+1)+(len(nums)-right)
        return min(front,back,both)
        